analyzers/rspec/cs/S3900_c#.html

```diff
@@ -1,5 +1,6 @@
-<p>A publicly accessible method can be called from anywhere, which means you should validate parameters to be within the expected constraints. In
-general, checking against <code>null</code> is recommended defensive programming.</p>
+<p>Methods declared as <code>public</code>, <code>protected</code>, or <code>protected internal</code> can be accessed from other assemblies, which
```

Comment:
When having this, the PR description should also have "Fixes #2643"

Reply:
I would probably create a separate PR for it, not to block this progress on wording around RSPEC